code/modules/jobs/job_types/roguetown/yeomen/mason.dm

```diff
@@ -20,16 +20,16 @@
 /datum/outfit/job/roguetown/mason/pre_equip(mob/living/carbon/human/H)
 	..()
 	if(H.mind)
-		H.mind.adjust_skillrank(/datum/skill/combat/axes, rand(1,3), TRUE)
-		H.mind.adjust_skillrank(/datum/skill/combat/maces, rand(1,3), TRUE)
-		H.mind.adjust_skillrank(/datum/skill/labor/lumberjacking, rand(2,3), TRUE)
-		H.mind.adjust_skillrank(/datum/skill/labor/mining, rand(2,3), TRUE)
-		H.mind.adjust_skillrank(/datum/skill/combat/wrestling, rand(1,3), TRUE)
-		H.mind.adjust_skillrank(/datum/skill/combat/unarmed, rand(1,3), TRUE)
-		H.mind.adjust_skillrank(/datum/skill/craft/crafting, rand(4,5), TRUE)
-		H.mind.adjust_skillrank(/datum/skill/craft/carpentry, rand(4,5), TRUE)
-		H.mind.adjust_skillrank(/datum/skill/craft/masonry, rand(4,5), TRUE)
-		H.mind.adjust_skillrank(/datum/skill/craft/engineering, rand(2,4), TRUE)
+		H.mind.adjust_skillrank(/datum/skill/labor/mining, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/axes, 2, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/maces, 2, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/labor/lumberjacking, 2, TRUE)
```

Comment:
you gave them lumberjacking instead of mining

Reply:
?

Reply:
its a mason no? Rockworker + miner